.golangci.yml

```diff
@@ -80,6 +80,35 @@ linters-settings:
       # - name: confusing-results
       - name: bool-literal-in-expr
       - name: atomic
+  depguard:
+    rules:
+      main:
+        list-mode: lax
+        deny:
+          - pkg: cosmossdk.io/errors
```

Comment:
can we add pkgerrors?  i know it's everywhere in core, but we can keep it out of here